main.go

```diff
@@ -15,158 +15,115 @@ import (
 	"github.com/AdguardTeam/dnsproxy/upstream"
 	"github.com/AdguardTeam/golibs/log"
 	"github.com/ameshkov/dnscrypt/v2"
+
+	"github.com/BurntSushi/toml"
 	goFlags "github.com/jessevdk/go-flags"
 	"gopkg.in/yaml.v3"
 )
 
 // Options represents console arguments
+
+type ConsoleArgs struct {
+	// Print DNSProxy version (just for the help)
+	Version  bool   `long:"version" description:"Prints the program version"`
+	TomlPath string `short:"c" description:"Path to the toml configuration file (optional)." default:"/etc/dnsproxy/dnsproxy.toml" ` //
+}
+
+// Program options. They will be taken from the toml file
+
 type Options struct {
 	// Log settings
 	// --
-
-	// Should we write
-	Verbose bool `short:"v" long:"verbose" description:"Verbose output (optional)" optional:"yes" optional-value:"true"`
-
-	// Path to a log file
-	LogOutput string `short:"o" long:"output" description:"Path to the log file. If not set, write to stdout." default:""`
+	Verbose   bool   // Should we write
```

Comment:
Please don't remove console args, this would make the new version console interface incompatible with previous versions.

Here's how the config file should work:

1. Add a new command-line option ("config-path")
2. If this option is specified, load options from the specified yaml file.
3. Other command-line options **override** the ones specified via the yaml file

Again, this should probably be discussed in https://github.com/AdguardTeam/dnsproxy/issues/57